website/static/js/fangorn.js

```diff
@@ -1012,6 +1171,465 @@ function scrollToFile(fileID) {
+                    ])
+            ]);
+    }
+}
+
+
+function _fangornResetToolbar () {
+    var tb = this;
+    tb.options.iconState.mode === 'bar';
+    tb.options.iconState = _defaultIconState();
+    m.redraw();
+}
+
+/**
+ * Toolbar icon templates
+ *
+ */
+function toolbarDismissIcon (){
+    var tb = this;
+    return m('.fangorn-toolbar-icon', {
+            onclick : function () { tb.options.iconState.mode = 'bar'; tb.resetFilter(); }
+        },
+        m('i.fa.fa-times')
+    );
+}
+ function searchIcon (){
+    var tb = this;
+    return m('.fangorn-toolbar-icon.text-info', {
+            'data-toggle' : 'tooltip',
+            'title':  'Switch to search view.',
+            'data-placement' : 'bottom',
+            onclick : function () {
+                tb.options.iconState.mode = 'search';
+                tb.filterText('');
+                m.redraw(true);
+
+            }
+        }, [
+        m('i.fa.fa-search'),
+        m('span.hidden-xs', 'Search')
+    ]);
+ }
+  function infoIcon (){
+    var tb = this;
+    return m('.fangorn-toolbar-icon.text-info', {
+            'data-toggle' : 'tooltip',
+            'title':  'Learn more about how to use the file browser.',
+            'data-placement' : 'bottom',
+            onclick : function () {
+                var mithrilContent = m('div', [
+                        m('h3.break-word', 'How to Use the File Browser'),
+                        m('p', 'Select Multiple Files: Use command or shift keys to select multiple files.'),
+                        m('p', 'Go to Files: Double click a file name to go to the file.'),
+                        m('p', 'Open Files in New Tab: Command + click a file name to open it in a new tab.'),
+
+                    ]);
+                var mithrilButtons = m('div', [
+                        m('span.tb-modal-btn', { 'class' : 'text-primary', onclick : function() { tb.modal.dismiss(); } }, 'Close'),
+                    ]);
+                tb.modal.update(mithrilContent, mithrilButtons);
+
+            }
+        }, [
+        m('i.fa.fa-info')
+    ]);
+ }
+ function cancelUploadsIcon (){
+    var tb = this;
+    return m('.fangorn-toolbar-icon.text-warning', {
+            'data-toggle' : 'tooltip',
+            'title':  'Cancel currently pending downloads.',
+            'data-placement' : 'bottom',
+
+            onclick : function () {cancelUploads.call(tb); }
+        }, [
+        m('i.fa.fa-times-circle'),
+        m('span.hidden-xs', 'Cancel All Uploads')
+    ]);
+ }
+ function deleteMultipleIcon (){
+    var tb = this;
+    return m('.fangorn-toolbar-icon.text-danger', {
+            'data-toggle' : 'tooltip',
+            'title':  'Delete all of the currently selected items.',
+            'data-placement' : 'bottom',
+            onclick : function (event) {
+                    var configOption = resolveconfigOption.call(tb, tb.multiselected[0], 'removeEvent', [event, tb.multiselected]); // jshint ignore:line
```

Comment:
For consistency with other callbacks, how about `onRemove` instead of `removeEvent`?